.circleci/config.yml

```diff
@@ -877,7 +877,7 @@ jobs:
       - run: mkdir -p test-results
       - run:
           name: Test installing packages on target systems
-          command: make -f dockerfiles/verify_packages/Makefile INSTALL_TYPE=<< parameters.install_type >> docker_compose_pull verify_centos_6
+          command: make -f dockerfiles/verify_packages/Makefile INSTALL_TYPE=<< parameters.install_type >> verify_centos_6
```

Comment:
`dockerfiles/verify_packages/docker-compose.yml` is [parameterized](https://github.com/DataDog/dd-trace-php/blob/b8c368e9cb42cd5bf1ed278ae3fa71c5ff64e47d/dockerfiles/verify_packages/docker-compose.yml#L21) so a generic pull without envs set is pointless